src/i18n/messages.json

```diff
@@ -74,6 +74,90 @@
         "description": "Globetrotter (covid permettendo), amante delle bibite gassate e climber. Amo le sfide. Mi piace studiare quello che mi interessa. Sono sintetico.",
         "website": "",
         "permalink": "nicpuppa"
+      },
+      "Agny": {
```

Comment:
There is a typo in the name, the correct one is Angy.

Reply:
done